device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/auth/IotHubSasTokenHardwareAuthenticationProvider.java

```diff
@@ -65,25 +65,14 @@ public IotHubSasTokenHardwareAuthenticationProvider(String hostname, String gate
     /**
      * Getter for SasToken. If the saved token has expired, this method shall renew it if possible
      *
-     * @param proactivelyRenew if true, this method will generate a fresh sas token even if the previously saved token
```

Comment:
so if we are repurposing this method entirely, where does this logic go? the proactive renewal approach that is.

Reply:
did we never use this?

Reply:
We only ever called this method with the same flags set, (true and true), so we can just get rid of the other cases